src/controller/CHIPDeviceController.cpp

```diff
@@ -2777,6 +2794,187 @@ CHIP_ERROR DeviceCommissioner::ParseICDInfo(ReadCommissioningInfo & info)
     return err;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+CHIP_ERROR DeviceCommissioner::ParseJFAdministratorInfo(ReadCommissioningInfo & info)
+{
+    using namespace JointFabricAdministrator::Attributes;
+    using namespace OperationalCredentials::Attributes;
+    ByteSpan rootKeySpan;
+
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    err = mAttributeCache->ForEachAttribute(JointFabricAdministrator::Id, [this, &info](const ConcreteAttributePath & path) {
+        using namespace chip::app::Clusters::JointFabricAdministrator::Attributes;
+        AdministratorFabricIndex::TypeInfo::DecodableType administratorFabricIndex;
+
+        VerifyOrReturnError(path.mAttributeId == AdministratorFabricIndex::Id, CHIP_NO_ERROR);
+        ReturnErrorOnFailure(this->mAttributeCache->Get<AdministratorFabricIndex::TypeInfo>(path, administratorFabricIndex));
+
+        if (!administratorFabricIndex.IsNull())
+        {
+            ChipLogProgress(Controller, "JCM: AdministratorFabricIndex: %d", administratorFabricIndex.Value());
+            info.JFAdministratorFabricIndex = administratorFabricIndex.Value();
+            info.JFAdminEndpointId          = path.mEndpointId;
```

Comment:
Can there be more than one JointFabricAdministrator cluster on the node?  Or is there at most one, but we just don't know which endpoint it will be on?

Reply:
Second case: there is a single JointFabricAdministrator cluster on the node but we don't know which endpoint it will be on.